### PR TITLE
Don't rename aarch64 architecture JIRA SB-3832

### PR DESCRIPTION
**Pull request by merezhuk**
Leave returned by uname -m aarch64 as is to conform name of a static binary archive for ARM platform on FTP site

### CODE REVIEW

**Comment by boratyng** [comment]
I see that in version 2.13.0 there is "x64-arm" in the package file name: https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.13.0/ncbi-blast-2.13.0+-x64-arm-linux.tar.gz, but in 2.14.1 it was changed to "aarch64": https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.14.1/ncbi-blast-2.14.1+-aarch64-linux.tar.gz.

I guess we should decide on one and stick to it for future releases. I am fine with either one.

**Review by christiam — APPROVED** [review]

**Comment by christiam** [comment]
I don't think there was much consensus/thought given when `x64-arm` was used and there's a symlink for it in the 2.13.0 release. We now have scripts that use `aarch64`, so I'd favor that.

**Review by boratyng — APPROVED** [review]

**Comment by merezhuk** [comment]
Ok thank you everybody! Problem was that internally in Dockerfile for aarch64 platform we used different naming schema only to retrieve  static ARM tarball from out FTP site and for 2.14.1 release it was broken.

Actual name right now on FTP is:
https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.14.1+-aarch64-linux.tar.gz
while in Dockerfile it was overeaten to be:
ftp://ftp.ncbi.nlm.nih.gov/blast/executables/LATEST/ncbi-blast-2.14.1+-x64-arm-linux.tar.gz 

We have script to create manifest for multi-arch docker and this part will not be changed.

Basically I'd like to have working dockerfile for ARM platform :)
